.doctrine-project.json

```diff
@@ -4,11 +4,17 @@
     "slug": "doctrine-migrations-bundle",
     "versions": [
         {
-            "name": "2.0",
+            "name": "2.1",
```

Comment:
master is 2.2-dev as we've got the 2.1.x branch already. This might need reflecting in composer.json, but the docs should account for it.

Reply:
Okay, I've updated the file. Please take a look.